README.md

```diff
@@ -140,13 +140,16 @@ properties of the Entity object detailed in the following table (as per `sensor.
 | y_axis | string |  | If 'secondary', displays using the secondary y-axis on the right.
 | fixed_value | boolean |  | Set to true to graph the entity's current state as a fixed value instead of graphing its state history.
 | smoothing | boolean |  | Override for a flag indicating whether to make graph line smooth.
+| scale_factor | number | 1 | Set a scale factor to use on the graph's value
```

Comment:
Please rename (back) to `value_multiplier`, as `scale_factor` will lead to more confusion